src/components/DotIndicatorMessage.js

```diff
@@ -71,14 +89,33 @@ function DotIndicatorMessage(props) {
                 />
             </View>
             <View style={styles.offlineFeedback.textContainer}>
-                {_.map(sortedMessages, (message, i) => (
-                    <Text
-                        key={i}
-                        style={[StyleUtils.getDotIndicatorTextStyles(isErrorMessage), ...props.textStyles]}
-                    >
-                        {message}
-                    </Text>
-                ))}
+                {_.map(sortedMessages, (message, i) =>
+                    isReceiptError(message) ? (
+                        <PressableWithoutFeedback
+                            key={i}
+                            accessibilityRole={CONST.ACCESSIBILITY_ROLE.LINK}
+                            onPress={() => {
+                                fileDownload(message.source, message.filename);
+                            }}
+                        >
+                            <Text
+                                key={i}
+                                style={styles.offlineFeedback.text}
+                            >
+                                <Text style={[styles.optionAlternateText, styles.textLabelSupporting]}>{Localize.translateLocal('iou.error.receiptFailureMessage')}</Text>
+                                <Text style={[styles.optionAlternateText, styles.textLabelSupporting, styles.link]}>{Localize.translateLocal('iou.error.saveFileMessage')}</Text>
+                                <Text style={[styles.optionAlternateText, styles.textLabelSupporting]}>{Localize.translateLocal('iou.error.loseFileMessage')}</Text>
+                            </Text>
```

Comment:
@Gonals The error text next to red dot indicators is meant to be red to be consistent with other error messages in the app. 

With your current change here, the error text will revert back to light-greyish green. 

![expensify-new-error-text-bug-introduced](https://github.com/Expensify/App/assets/79470910/6e4c7507-8865-4b6b-96e2-09c8ec1c6c7a)

I just worked on an [issue](https://github.com/Expensify/App/issues/29812) fixing this and just luckily stumbled on the regression from the video of another issue I was looking at.